### PR TITLE
specified which repository was intended

### PR DESCRIPTION
**Pull request by MirjamvT**

### CODE REVIEW

File: standards/coding-in-the-open.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ order: 1
 
 ## Measure
 
-* All source code for any used policy and software not used for fraud detection MUST be published and publicly accessible.
-* Any source code that is not currently in use, such as new versions, proposals or older versions SHOULD be published.
-* Contributors MUST never upload sensitive information regarding users, the organization or third parties to the repository, for example: configurations, usernames and passwords, public keys and other real credentials used in the production system.
+* All source code and used policy and software not used for fraud detection MUST be published and publicly accessible.
+* Any source code that is not currently in use, such as new versions, proposals or older versions SHOULD be published. 
+* Contributors MUST never upload sensitive information regarding users, the organization or third parties to the Public Code repository, for example: configurations, usernames and passwords, public keys and other real credentials used in the production system.
```
Comment (bvhme) [inline standards/coding-in-the-open.md]: This standard should be applicable to anyone, also to others, so specifying the repository as ours isn't necessary. Apart from that the user should never upload these things to any repository anyways.

**Comment by clausmullie** [comment]
It's asking for command line to resolve conflicts..

@bvhme or @ElenaFdR can you, at some point, show me how to do this? thanks! ^^

**Comment by bvhme** [comment]
FYI: this has also been solved already, the version in the develop is:

> All source code for any policy and software in use (unless used for fraud detection) MUST be published and publicly accessible.

If that version is better than this new one I'd recommend going with that.

**Comment by MirjamvT** [comment]
I like that sentence better

**Pull request closed without merging**